test-js/unit/pentaho/type/simple.serialization.Spec.js

```diff
@@ -180,35 +160,50 @@ define([
     });
 
         describe("values", function() {
-          describe("primitive format (returnFormattedValues: false)", function() {
+          describe("primitive format (omitFormatted: true)", function() {
             it("should return the primitive value", function() {
-              var spec = value.toSpec({returnFormattedValues: false, inlineTypeSpec: false});
+              var spec = value.toSpec({omitFormatted: true, omitRootType: true});
 
               expect(typeof spec).toBe(simple.name);
               expect(spec).toBe(simple.value);
             });
           });
 
-          describe("object format (returnFormattedValues: true)", function() {
+          describe("object format (omitFormatted: false)", function() {
             it("spec.v should contain the primitive value", function() {
-              var spec = value.toSpec({returnFormattedValues: true});
+              var spec = value.toSpec({omitFormatted: true});
 
               expect(typeof spec).toBe("object");
               expect(spec.v).toBeDefined();
-
               expect(typeof spec.v).toBe(simple.name);
               expect(spec.v).toBe(simple.value);
+              expect(spec._).toBeDefined();
+              expect(spec._.id).toBe(SimpleClass.type.toSpec().id);
             });
           });
         });
 
         describe("inline type spec", function() {
           it("should inline pentaho/type/" + simple.name + " type spec", function() {
```

Comment:
?....This simple.name is the name of the simplerTypes object we are looping over in this test...